tests/e2e/default/targets/target-aggregates.wdio-spec.js

```diff
@@ -367,9 +372,8 @@ describe('Target aggregates', () => {
       expect(await contactsPage.getContactInfoName()).to.equal('Clarissa');
       // assert that the activity card exists and has the right fields.
       expect(await contactsPage.getContactCardTitle()).to.equal('Activity this month');
-      validateCardField('Last updated', 'yesterday Clarissa');
-      validateCardField('what a target!', '40');
-      validateCardField('the most target', '50%');
```

Comment:
The reason the tests were failing silently is these validations weren't being `await`ed. As soon as I added the `await` the test failed.